proposals/stage-0/SECURITY_REPORTING/README.md

```diff
@@ -0,0 +1,60 @@
+# My New Proposal
+>  Stage 0
+
+## Champion
+
+Marcin Hoppe (@marcin_hoppe)
+
+## Description
+
+Currently OpenJS Foundation projects do not have consistently documented standards for responsible reporting and disclosure of security vulnerabilities. This proposal aligns OpenJS Foundation projects with industry best practices in this space.
+
+See also:
+- [Security reporting in OpenJS Foundation projects](https://gist.github.com/MarcinHoppe/b13a870770522c31a8386ada48b2e40f)
+- [Guidelines for responsible reporting and disclosure of security vulnerabilities](https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/security-guidelines.md)
+
+## Required Resources
+
+- Generic security policy template that projects can easily adopt
+- Optionally: vulnerability disclosure platform (VDP)
+
+## Who would be responsible?
+
+The responsibility for adoption of security reporting standards would be jointly owned by the projects and the CPC.
+
+## How would success be measured?
+
+- 100% of Impact, Growth, and At-Large projects have:
+    - Security policy on GitHub
+    - Vulnerability reporting mechanism (email or VDP)
+- 100% projects graduating from Incubation have a security policy on GitHub
+
+## Why this proposal is important
+
+Open source projects that can responsibly accept security reports and respond with a coordinated vulnerability disclosure have a higher chance of being treated as trustworthy by adopters.
+
+## Unresolved Question
+
+- Do we leave it to projects to coordinate disclosure?
+    - Some projects are already prepared, have infrastructure and processes in place. Others don't.
```

Comment:
I believe it was mentioned in today's call, but having a playbook projects can follow would be superhelpful.

Reply:
@molant Thanks for reminding me about this. I added the playbook to the list of resources.